javascript/app-model-exposure/src/utils/results.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+function isStringNan(n: string | number | null): boolean {
+	return typeof n === 'string' && n === 'nan'
+}
+
+export function nanToNull(result: Record<string, any>) {
```

Comment:
This function is actually targeting a very specific property that isn't expressed in the name. It might be more appropriate to either rename it or include a property name string as one of the args, e.g:

```
(result: ..., property: string) {
...
if (result.hasOwnProperty(property))
...etc
```